### PR TITLE
Start content style guide

### PR DESCRIPTION
**Pull request by missmatsuko**
Very very rough draft!

Pulled some content from CONTRIBUTING.md and [this comment by svinkle](https://github.com/a11yproject/a11yproject.com/issues/509#issuecomment-325069162).

### CODE REVIEW

**Comment by ericwbailey** [comment]
I think this is an excellent start, thank you! I will be submitting a PR shortly that will build off of this, with updates to the README and CONTRIBUTING docs, so I'd like to get this merged in first to reference it. @svinkle, @scottaohara - does it look good to you?

And thank you again for your work on the site, @missmatsuko!

**Comment by missmatsuko** [comment]
Thanks! Glad to have been able to help!

**Comment by svinkle** [comment]
So much love for this! Thank you, @missmatsuko. 🙌

Question about the formatting and tone sections:  are these considered complete? Or are we looking to merge this is as a "first draft" type deal?

File: CONTENT_STYLE_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Complexity
+- **Short** - Aim for a timed reading length of approximately two minutes.
+- **Focused** - Keep it digestible and to a single topic. Articles that span multiple areas and topics are better broken up.
+- **Accessible** - Strive to use plain language, avoid jargon if possible, and explain complicated concepts.
+
+**Resources:**
+- [Hemingway App](http://www.hemingwayapp.com/)
+
+## Tone
+Tone of voice tips. Professional and clear.
```
Comment (scottaohara) [inline CONTENT_STYLE_GUIDE.md]: to merge this in, we either need to flesh this section out a bit, or remove it until we have more than just this placeholder.

File: CONTENT_STYLE_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Spelling
+
+### American, UK, or Canadian Spelling
+For better or for worse, American English is generally considered to be the standard language for web developers. Let's try to use American English spelling for content on untranslated pages for consistency and clarity.
+
+### Spell-check
+Check your spelling before submitting content. Word processors usually have spell-check built-in. There are also many spell-checking websites and browser extensions.
+
+## Formatting
+Markdown tips. Heading levels. Templates?
```
Comment (scottaohara) [inline CONTENT_STYLE_GUIDE.md]: to merge this in as is, we should remove "heading levels. templates?"

**Pull request merged**